Documentation/specs/runtime-configuration-file.md

```diff
@@ -25,8 +25,15 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
 ```json
 {
     "runtimeOptions": {
-        "gcServer": true,
-        "gcConcurrent": false,
+
+        "configProperties": {
+            "System.GC.Server": true,
```

Comment:
I don't want people to use this as a general config abstraction because it isn't that. More data about the naming scheme and why it's relevant would be good